src/sidebar/services/annotations-exporter.ts

```diff
@@ -39,4 +46,50 @@ export class AnnotationsExporter {
       ) as APIAnnotationData[],
     };
   }
+
+  buildTextExportContent(
+    annotations: APIAnnotationData[],
+    groupName = '',
+    /* istanbul ignore next - test seam */
+    now = new Date(),
+  ): string {
+    const [firstAnnotation] = annotations;
+    if (!firstAnnotation) {
+      throw new Error('No annotations to export');
+    }
+
+    const { uri, title } = documentMetadata(firstAnnotation);
+    const uniqueUsers = [
+      ...new Set(
+        annotations
+          .map(annotation => username(annotation.user))
+          .filter(Boolean),
+      ),
+    ];
+
+    const annotationsText = annotations
+      .map(
+        (annotation, index) =>
+          trimAndDedent`
+          Annotation ${index + 1}:
+          ${annotation.created}
+          ${quote(annotation)}
+          ${username(annotation.user)}
+          "${annotation.text}"
```

Comment:
In the example given inhttps://docs.google.com/document/d/1Xn6AazKkpUy85VUaj9v_utasexk8lIqP_1zveILq1So/edit#heading=h.g853pir10o5, the annotation _quote_ has double-quotes around it, not the annotation body.

Reply:
Good catch